helix-core/src/main/java/org/apache/helix/model/LiveInstance.java

```diff
@@ -130,6 +140,22 @@ public void setHelixVersion(String helixVersion) {
     _record.setSimpleField(LiveInstanceProperty.HELIX_VERSION.toString(), helixVersion);
   }
 
+  /**
+   * Gets the live instance's status.
```

Comment:
nit, please comment about the return value, it would be null if there is no status specified.